docs/pages/access-controls/access-requests/role-requests.mdx

```diff
@@ -260,7 +260,14 @@ $ tctl request approve --roles=role-1,role-3 --reason='Approved, but not role-2
   older versions of Teleport, since only the issuing Auth Server needs any particular
   knowledge of the feature.
 
+## Next Steps
 
-## Integrating with an External Tool
+### Integrating with an External Tool
 (!docs/pages/includes/access-request-integrations.mdx!)
 
+### Using TTLs with Access Requests
+
+`tsh request new` support flags to control TTLs for the request and
```

Comment:
There's no other mention of `tsh request new` in this guide, so I think we'll need some context for readers to understand what `tsh request new` is and when you would use it. 

If it's possible to add a TTL to Access Requests when using the `tsh login` command with the `--request-roles` flag, I think we can add a note about the TTL when we include an example of the `tsh login` command earlier in the guide.